### ISSUE TITLE
Blender 3.0 Failure

### ISSUE THREAD

**Issue by TimMensch** [BabylonJS/BlenderExporter#54]
Not sure if you guys are supporting Blender 3.0.0, but having installed from the Windows Store, Blender auto-updated on me and now I have 3.0.0 installed.

In any event, a scene that I was previously able to export is suddenly throwing an error:

<img width="622" alt="blender_2021-12-10_15-06-47" src="https://user-images.githubusercontent.com/530873/145647780-171e8130-b6b0-483a-92ca-2ea528249f03.png">

The relevant bits are:

```
AttributeError: 'RenderSettings" object has no attribute 'tile_x'
```

This really feels like a change between 2.9.x and 3.0.0 in Blender, but please let me know if there's something that I'm likely to need to change in my scene to fix this.

**Comment by Palmer-JC** [comment]
Had too many change / fix / change episodes with 2.80.  Only the LTS
versions (last was 2.93) are going to be supported.  Have gotten reports
that 3.0 was working, but looks like material baking is not.

Baking is not really the best outcome anyway.  You might look at the export
log file and see the material being worked on & the node requiring the
baking.  Perhaps you might do it another way, Principled recommended.  Or
drop back to 2.93.

Will leave this open till 3.03 LTS goes to beta, and work begins.  Thanks
for reporting.

**Comment by impactcolor** [comment]
Same issue.... AttributeError: 'RenderSettings' object has no attribute 'tile-x'
Is this something relating to the render method?

**Comment by Palmer-JC** [comment]
Basically, at the API level they made changes which are not back compatible.

**Comment by larjohn** [comment]
Is there an alternative to baking for things like bump maps (for instance those coming from MakeHuman clothing)?

**Comment by Palmer-JC** [comment]
Sure, bake it yourself.  Not that hard to do it & worth knowing.  Also, just use 2.93.  You can have both installed.

**Comment by echan00** [comment]
also having the same problem, going to try 2.93 instead

**Comment by itssource** [comment]
Is this project deprecated for Blender 3.x? I see references to random posts here and there on the Internet about using this repository, but it seems development stopped at 2.9. What's the best practice today? Export from Blender for now through menu item glTF 2.0 (.glb/.gltf)?

**Comment by Palmer-JC** [comment]
Yes, there will be a version for 3.3 LTS.  The intermediate Blender
versions are not published for.  3.3 is in alpha.  The only known bug,
requiring the exporter to bake textures, has been fixed.  Not sure how much
else will be new / changed yet, though.

**Comment by itssource** [comment]
Okay, thanks Jeff.

**Comment by ElvisWong** [comment]
I got the same issue when exporting my blender scene using babylonjs exporter. Here's the error logs
```
Exporter version: 2.93.5, Blender version: 3.1.2
========= Conversion from Blender to Babylon.js =========
	Scene settings used :
		Inline textures     :  false
		Material Type       :  PBR
		Positions Precision :  4
		Normals Precision   :  3
		UVs Precision       :  3
		Vert Color Precision:  3
		Mat Weight Precision:  2
		Keep Z-up r-handed  :  no
		Texture directory   :  /Users/[xxx]/Documents/blender/[xxx]/
	Python World class constructor completed
	processing begun of camera (UniversalCamera):  Camera
	processing begun of mesh:  Room
		processing begun of material:  Wall Light
		processing begun of material:  Wall Base
			WARNING: unsupported node type(ShaderNodeMixRGB) will trigger baking
			WARNING: unsupported node type(ShaderNodeGamma) will trigger baking
			WARNING: Metal channel baking required, but not possible, ignored
			WARNING: Roughness channel baking required, but not possible, ignored
			WARNING: Refraction channel baking required, but not possible, ignored
========= An error was encountered =========
  File "/Users/[xxx]/Library/Application Support/Blender/3.1/scripts/addons/babylon_js/json_exporter.py", line 116, in execute
    mesh = Mesh(object, scene, self)
  File "/Users/[xxx]/Library/Application Support/Blender/3.1/scripts/addons/babylon_js/mesh.py", line 150, in __init__
    recipe = BakingRecipe(bpyMesh, exporter)
  File "/Users/[xxx]/Library/Application Support/Blender/3.1/scripts/addons/babylon_js/materials/baking_recipe.py", line 105, in __init__
    self.bakedMaterial.bake(bpyMesh, self)
  File "/Users/[xxx]/Library/Application Support/Blender/3.1/scripts/addons/babylon_js/materials/material.py", line 145, in bake
    tileXHold = render.tile_x
ERROR:  'RenderSettings' object has no attribute 'tile_x'
========= end of processing =========
elapsed time:  0 min, 0.0084 secs
```

**Comment by Palmer-JC** [comment]
@ElvisWong, you may not use this exporter with anything that results in the material being required to bake, in a version of Blender newer than 2.93.

Your material 'Wall Base' has 2 nodes which have no equivalent in BABYLON PBR.  MixRGB node might be added, but not Gamma.  Baking in Blender also means there is going to be a texture required.  Much better to use something like a Principled node for your materials, which maps much better, thus avoiding the bake / texture at run time.
